docs/guides/chip_tool_guide.md

```diff
@@ -861,17 +865,50 @@ In this command:
 -   _<endpoint_id\>_ is the ID of the endpoint on which the `binding` cluster is
     implemented.
 
+##### Running `TestClusters` test
+
+Complete the following steps to
+[run one test from the test suite](#running-a-test-suite-against-a-paired-peer-device):
+
+1. Clean the initialization of state using the following command:
+    ```
+    rm -fr /tmp/chip_*
+    ```
+1. In a shell window, start the DUT device:
+    ```
+    ./out/debug/standalone/chip-all-clusters-app
+    ```
+1. In a second shell window, pair the DUT with the CHIP Tool:
+    ```
+    ./out/debug/standalone/chip-tool pairing onnetwork 333221 20202021
```

Comment:
This is actually fairly error-prone and we should not steer people toward this.  People should pair using the device's onboarding payload.

Reply:
I moved this part from the old line 651. Not really my call to remove this, so should this be tracked as a separate issue?

Reply:
Ah, I had missed this just being moved.  In that case, yes, separate issue.  Just drop the link to the issue in this conversation and resolve the conversation, please.

Reply:
Reported issue: https://github.com/project-chip/connectedhomeip/issues/25780